docs/data-ingestion/README.md

```diff
@@ -1,5 +1,5 @@
 In This Folder you will find instructions for sending data to VictoriaMetrics from a variety of platforms.
-If your tool is not listed it is likely you can ingest your data into VictoriaMetrics using one of the protocols listed in our [Prominent features]({{< ref "/Single-server-VictoriaMetrics.md#prominent-features" >}}) section.
+If your tool is not listed it is likely you can ingest your data into VictoriaMetrics using one of the protocols listed in our [Prominent features](../#prominent-features) section.
```

Comment:
@tiny-pangolin  it is recommended using absolute links in documentation - see this guideline https://docs.victoriametrics.com/#documentation